gnome-initial-setup/eos-demo-mode

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# eos-setup-demo-user - configure demo user account
+# Copyright (C) 2016-2017 Endless Mobile, Ltd.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import argparse
+import glob
+import json
+import logging
+import os
+import subprocess
+import sys
+
+from gi.repository import GLib, Gio
+
+log = logging.getLogger(sys.argv[0])
+
+ICON_GRID_DIR = '/var/lib/eos-image-defaults/icon-grid'
+
+DESKTOP_GRID_ID = 'desktop'
+
+GOOGLE_CHROME = 'google-chrome.desktop'
+CHROMIUM_BROWSER = 'chromium-browser.desktop'
+
+GDM_DAEMON_CONF = '/etc/gdm3/daemon.conf'
+
+
+def dq(s):
+    return '"{}"'.format(s)
+
+
+def remove_chrome_from_icon_grids():
```

Comment:
I'm quite confused as to why this is needed (and it overwrites system files in `/var` which is pretty unusual); in the settings code we do the same per-user instead of on the system files -- why is that not enough?